lib/linguist/languages.yml

```diff
@@ -2381,6 +2381,15 @@ Ignore List:
   codemirror_mode: shell
   codemirror_mime_type: text/x-sh
   language_id: 74444240
+ImageJ Macro:
+  type: programing
```

Comment:
Typo there, causing a failed test: `invalid type: programing (ArgumentError)`. Should be "programming" with 2 Ms.

Reply:
Thanks Nixinova. I am one step closer.